.github/workflows/shisen_ci.yml

```diff
@@ -8,17 +8,21 @@ on:
       - master
       
 jobs:
-  build:
-
+  build-and-deploy:
     runs-on: ubuntu-latest
-    name : Build the Project
-
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-node@v2
-      with:
-        node-version: '14'
-    - run: npm install
-    - run: npm run build
-    - run: npm test
+      - name: Checkout 
+        uses: actions/checkout@v2
+
+      - name: Install and Build 
+        run: |
+          npm install
+          npm run build
+          npm test
+
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@4.1.0
+        with:
+          branch: gh-pages
+          folder: build 
```

Comment:
Sorry my bad, the folder should be `dist` instead of `build`.
```
  File                                      Size             Gzipped

  dist/js/chunk-vendors.0ef12dcd.js         240.61 KiB       80.96 KiB
  dist/js/chunk-4e4cae24.65ff97a1.js        129.18 KiB       36.13 KiB
  dist/js/app.971d6dc9.js                   7.22 KiB         3.00 KiB
  dist/service-worker.js                    1.05 KiB         0.61 KiB
  dist/precache-manifest.79dd3dec06da51a    0.86 KiB         0.34 KiB
  a40b6dc33739531c9.js
  dist/css/chunk-vendors.8c99960f.css       300.32 KiB       35.72 KiB
  dist/css/chunk-4e4cae24.e839ede7.css      68.75 KiB        8.52 KiB
```

Reply:
will be changed shortly